FirstProject/assets/login.less

```diff
@@ -0,0 +1,73 @@
+//Login.less
+
+body {
+  background-color:#fafafa;
+}
+
+header {
+  background: url('../img/feedback-banner.jpg') center center no-repeat;
+  background-size: cover;
+  height: 600px;
+
+  .grid {
+    height: 100%;
+  }
+}
+
+.login-form {
+  background-color: #ffffff;
+  align-self: center;
+  text-align: center;
+  padding-bottom: 30px;
+
+  .login-form-header {
+    line-height: 60px;
+    background-color: #03A9F4;
+    color: #ffffff;
+    font-weight: bold ;
+    text-align: center;
+  }
+
+  .login-form-content  {
+    padding:15px;
+    input {
+      width: 100%;
+      border: none;
+      border-bottom: 1px solid #cccccc;
+      margin-top: 25px;
+    }
+  }
+
+  .login-form-submit {
+
+  }
+}
+
+
+.login-section-1 {
```

Comment:
Vali used this class.. 
When we have for example 3-4 sections, we can use .login-section-1,.logni-section-2 and so on.